Makefile

```diff
@@ -44,8 +46,17 @@ help: ## Display this help.
 
 ##@ Development
 
+.PHONY: manifests-update
+manifests-update:
+	wget -O - https://raw.githubusercontent.com/kserve/kserve/$(KSERVE_MANIFESTS_REVISION)/config/crd/full/serving.kserve.io_inferencegraphs.yaml \
+		| tail -n +2 > config/crd/external/serving.kserve.io_inferencegraphs.yaml
+	wget -O - https://raw.githubusercontent.com/kserve/kserve/$(KSERVE_MANIFESTS_REVISION)/config/crd/full/serving.kserve.io_inferenceservices.yaml \
+		| tail -n +2 > config/crd/external/serving.kserve.io_inferenceservices.yaml
+	wget -O - https://raw.githubusercontent.com/kserve/kserve/$(KSERVE_MANIFESTS_REVISION)/config/crd/full/serving.kserve.io_servingruntimes.yaml \
+		| tail -n +2 > config/crd/external/serving.kserve.io_servingruntimes.yaml
```

Comment:
I like this thinking a lot. Automate all the things! It deserves its own PR in my eyes. It's an improvement to a build process which is out of scope of simply bumping KServe :)

Another thing to note is that this will be executed every time, regardless of the actual changes to the revision we rely on. This might (or might not) be a problem if we ever move it to Konflux and use "hermetic builds". From the top of my head some "crd.sum" check of some sort might help (or even to rely on built-in Make checks with some guard files instead).

Reply:
hermetic - offline

Reply:
> Another thing to note is that this will be executed every time, regardless of the actual changes to the revision we rely on. This might (or might not) be a problem if we ever move it to Konflux and use "hermetic builds". From the top of my head some "crd.sum" check of some sort might help (or even to rely on built-in Make checks with some guard files instead).

I think we can follow up at optimizing when, I believe this one is already an improvement, at least to have a consistent revision from which CRDs are taken, currently they are all based on different upstream revisions and it's too easy to miss an update

Reply:
https://github.com/opendatahub-io/odh-model-controller/pull/450 (also with the guarding)

Reply:
I was thinking of the same - let's bring the automation/unification first, as it's valuable on its own, and then we can improve.